src/utils/illustration.js

```diff
@@ -160,10 +160,25 @@ const data = [{
 	strings: [
 		'Meeting',
 		'Meet',
+		'Planning',
+		'Pointing',
+		'Retrospective', 'Retro',
+		'Review',
+		'Stand-up', 'Standup',
 		// TRANSLATORS This string is used for matching the event title to an illustration
 		t('calendar', 'Meeting'),
 		// TRANSLATORS This string is used for matching the event title to an illustration
 		t('calendar', 'Meet'),
+		// TRANSLATORS This string is used for matching the event title to an illustration
+		t('calendar', 'Planning'),
+		// TRANSLATORS This string is used for matching the event title to an illustration
+		t('calendar', 'Pointing'),
+		// TRANSLATORS This string is used for matching the event title to an illustration
+		t('calendar', 'Retrospective'),
+		// TRANSLATORS This string is used for matching the event title to an illustration
+		t('calendar', 'Review'),
+		// TRANSLATORS This string is used for matching the event title to an illustration
+		t('calendar', 'Stand-up'),
```

Comment:
I'm not sure if this is a bit too ambiguous. It will also match "Stand-up comedy" and "Stand-up paddling"

Reply:
Agreed. Please remove this one for now @shtrom

Reply:
Ah, that's unfortunate, that was the one that motivated to make those changes in the first place. Shall I also remove “standup” above?

Reply:
Yes please

Reply:
Maybe `wakeup` is more unambiguous and still fits.

Reply:
@raimund-schluessler This is not supposed to mean wakeup. It's supposed to be Standup as in agile software development: https://en.wikipedia.org/wiki/Stand-up_meeting

Reply:
Ah, I see, then `stand-up` *really* is ambiguous 😄

Reply:
Done. Though I wonder how we could handle those a bit better. Perhaps some per-calendar or per-user setting to allow disambiguating? Well beyond this PR, but I guess that would be a way to make it work.